specs/core/0_beacon-chain.md

```diff
@@ -846,6 +847,108 @@ def get_next_epoch_committee_count(state: BeaconState) -> int:
     return get_epoch_committee_count(len(next_active_validators))
 ```
 
+### `get_crosslink_committees`
+
+```python
+def get_crosslink_committees(state: BeaconState,
+                             seed: Bytes32,
+                             shuffling_epoch: Epoch,
+                             slot: Slot,
+                             start_shard: Shard,
+                             committees_per_epoch: int) -> List[Tuple[List[ValidatorIndex], Shard]]:
+    offset = slot % SLOTS_PER_EPOCH
```

Comment:
should we add an assert to check `shuffling_epoch` out of bound condition?

`assert get_previous_epoch(state)  <= shuffling_epoch <= get_current_epoch(state)+1`